packages/api/core/src/util/install-dependencies.ts

```diff
@@ -14,25 +14,38 @@ export enum DepVersionRestriction {
   RANGE = 'RANGE',
 }
 
+/**
+ *  To install the specified packages as devDependencies
+ * `npm add` and `pnpm add` commands use `--save-dev` option
+ * `yarn add` command use `--dev` option
+ */
+export const getInstallDevDepsOption = (): string => (isNpm() || isPnpm() ? '--save-dev' : '--dev');
```

Comment:
issue: I think all 3 manager support the `-D` shorthand syntax so I don't think we need this extra logic to differentiate between flags.